modules/@ergonode/ui/src/components/StaticImage/StaticImage.vue

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<template>
+    <img
+        ref="img"
+        :style="styles"
+        :class="classes">
```

Comment:
nie powinno być '/' na koncu ?

Reply:
img is called as Void element, doesnt require to close it.